test.txt

```diff
@@ -0,0 +1,57 @@
+CCExtractor version (using the --version parameter preferably) : **0.87**
```

Comment:
This file likely shouldn't be in here? :)

Reply:
Removed.